Overview.src.html

```diff
@@ -6243,7 +6243,10 @@ <h4 id="normalizing-a-sequence-of-keyframes">Normalizing a sequence of keyframes
 section></a> is performed before this operation is
 initiated.
 
-1.  If <var>frames</var> is a single {{Keyframe}} object, replaces
+1.  If <var>frames</var> is a {{SharedKeyframeList}} object, then
+    replace <var>frames</var> with the {{Keyframe}} or
+    sequence used to construct the {{SharedKeyframeList}}.
```

Comment:
No, I think we want to resolve to the list as part of normalization. There's a dissociation between this list and the computed keyframes which can be extracted.

If we expose getSpecifiedFrames (or whatever) then we'd want to return the SharedKeyframeList there, I think.